eq-author/src/App/publish/PublishPage.test.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
```

Comment:
Same as above, this was merged in the other day. Looks like a rebase has gone a bit weird.